src/createRuby.ts

```diff
@@ -66,10 +66,13 @@ export default async function createRuby() {
   // files to make it work. I'm not sure why I need to explicitly require
   // did_you_mean here, but it doesn't work without it.
   ruby.eval(`
+    require "rubygems"
```

Comment:
Is this require and the `pp` one necessary? I hope they aren't?

Reply:
Unfortunately, they are. Without them, we get `unutilized Gem lib/syntax_tree/formatter.rb:10` and `unutilized PP` in the browser.